src/test/java/edu/kis/powp/jobs2d/events/SelectTestFigureOptionListener.java

```diff
@@ -23,6 +24,15 @@ public void actionPerformed(ActionEvent e) {
 			case "Figure Joe 2":
 				FiguresJoe.figureScript2(driverManager.getCurrentDriver());
 				break;
+			case "Command test":
+				TestCommand.customTest(driverManager.getCurrentDriver());
+				break;
+			case "Rectangle factory":
+				TestCommand.rectangleTest(driverManager.getCurrentDriver());
+				break;
+			case "Triangle factory":
```

Comment:
Menu action should not depend on its caption. Enum or separate listeners would be better. At least you should consider constants.